bin/solr-configs-reset

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+cp /opt/solr/configs/funcake-oai-solr/solrconfig.xml /var/solr/data/funcake/
```

Comment:
@dkinzer Would this be funcake-oai-solr or just funcake-solr for this script?

Reply:
Good catch. This needs to be funcake